PL2/PL2-gui/src/main/java/nl/tudelft/pl2016gr2/gui/view/graph/DrawGraph.java

```diff
@@ -87,36 +86,35 @@ private void drawEdges(Pane pane, HashMap<Integer, Node> nodes,
         pane.getChildren().add(edge);
         edge.toBack();
         edge.setSmooth(true);
-
-        //        QuadCurve curve = new QuadCurve();
-        //
-        //        curve.setFill(Color.TRANSPARENT);
-        //        curve.setStroke(Color.BLACK);
-        //        curve.setStrokeWidth(2.0d);
-        //
-        //        fromCircle.centerXProperty().addListener((observable, oldValue, newValue) -> {
-        //          curve.startXProperty().setValue(newValue);
-        //          bendCurve(curve, fromCircle, toCircle);
-        //        });
-        //        fromCircle.centerYProperty().addListener((observable, oldValue, newValue) -> {
-        //          curve.startYProperty().setValue(newValue);
-        //          bendCurve(curve, fromCircle, toCircle);
-        //        });
-        //        toCircle.centerXProperty().addListener((observable, oldValue, newValue) -> {
-        //          curve.endXProperty().setValue(newValue);
-        //          bendCurve(curve, fromCircle, toCircle);
-        //        });
-        //        toCircle.centerYProperty().addListener((observable, oldValue, newValue) -> {
-        //          curve.endYProperty().setValue(newValue);
-        //          bendCurve(curve, fromCircle, toCircle);
-        //        });
-        //
-        //        pane.getChildren().add(curve);
-        //        curve.toBack();
       }
     });
   }
 
+  //        QuadCurve curve = new QuadCurve();
```

Comment:
This was added during pull request #27. I didn't really agree with merging this code, since it is not usefull yet (as I had also stated in my last comment on that pull request), since the lines were just bend arbitrarily and didn't avoid collision, so I just commented the code (and moved it out of the method to make the method smaller, which maybe caused the git blame to be incorrect?).